lib/mappers/maryland_mapper.py

```diff
@@ -11,7 +11,7 @@ def __init__(self, provider_data):
     def map_collection(self):
         collection = getprop(self.provider_data, "collection/title", True)
         if collection:
-            self.update_source_resource({"isPartOf": collection})
+            self.update_source_resource({"collection": {"title": collection}})
```

Comment:
should the collection dict include `id`, `@id`, and `description` properties even if they are empty?